x/quarantine/spec/04_events.md

```diff
@@ -3,30 +3,31 @@
 The `x/quarantine` module emits the following events.
 
 <!-- TOC -->
-  - [EventOptIn](#eventoptin)
-  - [EventOptOut](#eventoptout)
-  - [EventFundsQuarantined](#eventfundsquarantined)
-  - [EventFundsReleased](#eventfundsreleased)
+
+- [EventOptIn](#eventoptin)
+- [EventOptOut](#eventoptout)
+- [EventFundsQuarantined](#eventfundsquarantined)
+- [EventFundsReleased](#eventfundsreleased)
```

Comment:
These TOCs are auto-generated using `make update-tocs`. This change will be undone the next time someone runs that.

Reply:
No worries, the change was automatically made by the formatter in vscode

Reply:
This change needs to be undone before we merge. Otherwise, it'll just get included in some other unrelated PR when some spec documentation is updated.

Reply:
Updated